packages/gatsby-theme-newrelic/src/components/Link.js

```diff
@@ -31,6 +32,29 @@ const Link = ({ to, onClick, ...props }) => {
     }
   `);
 
+  const properties =
+    typeof instrumentation === 'string'
+      ? { customProp: instrumentation }
+      : instrumentation;
+
+  const handleClick = useInstrumentedHandler(onClick, {
+    actionName: 'externalLink_click',
+    href: to,
+    ...properties,
+  });
+
+  const link = isNewRelic(to) ? localizeExternalLink({ link: to, locale }) : to;
+
+  const trackSignUp = (event) => {
+    handleClick();
+    tessen &&
```

Comment:
Since you're pulling `tessen` from `useTessen`, no need to check if its defined here. `tessen` will always be defined, even if its not installed on the app. The [`createTessen` helper](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/utils/createTessen.js) checks to make sure `window.Tessen` is available and will result in a no-op if its not defined. Since `tessen` will always be defined here, the `track` will always get evaluated. Could you remove this since its unnecessary?

Reply:
yep! I added it in because one of the tests was resulting in a `tessen is not defined`